### ISSUE TITLE
GitHub / Intro to sprinting

### ISSUE THREAD

**Issue by mblayman** [python-frederick/talks#38]
This is the workshop used at the beginning of the PyCon US sprints to introduce people to Git and prepare them to work on open source projects.

**Comment by mblayman** [comment]
Here is the repo used for the workshop: https://github.com/chalmerlowe/intro_to_sprinting

**Comment by chalmerlowe** [comment]
@mblayman are you considering using the workshop at Python Frederick?
IF yes, we would love to get feedback on how it goes.
if you have questions, we would love to provide support.

FYI: This workshop will also be used at DjangoCon in October? 2018, so we are aiming to do some upgrades/cleanup in advance of that.

Overall, the workshop (in various forms) has been used at Pycon (twice), PyOhio (once), a meetup in Tennessee, and multiple times via our meetup in Hawaii (PyHawaii).

**Comment by mblayman** [comment]
@chalmerlowe yes, the plan is to use the workshop at some point this year. I attended the workshop at PyCon to observe what it was like. As an experienced developer, I appreciated the structure that it offered for new people.

I'll probably run the workshop at one of our monthly events. Thanks for putting it together!
